marketpulse/main/views.py

```diff
@@ -78,3 +79,13 @@ def activities(request):
                   {'user': request.user,
                    'total_contributions': total_contributions,
                    'percent': percent})
+
+
+def manifest(request):
+    import os
+    manifest = os.path.join(settings.BASE_DIR, 'static/manifest.webapp')
```

Comment:
This way we are rendering contribute.json [1] on Mozillians.org.

[1] https://github.com/mozilla/mozillians/pull/1143/files